macros/schema_tests/table_shape/expect_table_columns_not_to_contain_set.sql

```diff
@@ -0,0 +1,29 @@
+{%- test expect_table_columns_not_to_contain_set(model, column_list, transform="upper") -%}
```

Comment:
To match the other `*_not_*` tests, I think `expect_table_columns_to_not_contain_set` would be a better name. See: `expect_column_values_to_not_be_in_set`

Reply:
updated

Reply:
Thanks @clausherther , I may need help with the integration tests.  There is no data_test.csv file to seed when I run dbt seed from the integration_tests folder